content/ember-released-5-1.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Ember 5.1 Released
+authors:
+  - the-ember-learning-team
+date: 2023-06-23T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.1 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 5.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
```

Comment:
I am working on a good, detailed write-up to go here for our "Released" TS types here—will have it up today or Monday!

Reply:
```suggestion
Ember.js is the core framework for building ambitious web applications.

The headlining feature of this release is stable [TypeScript](https://www.typescriptlang.org) support! 🎉 The framework now publishes TypeScript types built from Ember's own source code, with Ember's [strong Semantic Versioning commitment](https://emberjs.com/releases/) baked in—bringing benefits for both TypeScript and JavaScript users. For details, check out [the accompanying blog post][stable-ember-types].

[stable-ember-types]: https://blog.emberjs.com/stable-typescript-types-in-ember-5-1
```

Note that this means we need to merge and publish the two posts at the same time so we avoid the link being dead!